warehouse/api/grpc.go

```diff
@@ -969,3 +969,32 @@ func statsInterceptor(statsFactory stats.Stats) grpc.UnaryServerInterceptor {
 		return res, err
 	}
 }
+
+func (g *GRPC) GetFirstAbortedUploadsInContinuousAborts(
+	ctx context.Context,
+	request *proto.FirstAbortedUploadsInContinuousAbortsRequest,
+) (*proto.FirstAbortedUploadsInContinuousAbortsResponse, error) {
+	g.logger.Infow(
+		"Getting first aborted uploads in a series of continuous aborts",
+		lf.WorkspaceID, request.WorkspaceId,
+	)
```

Comment:
nitpic: We can use structured logging in here,
```suggestion
	g.logger.Infon(
		"Getting first aborted uploads in a series of continuous aborts",
		obskit.WorkspaceID(request.WorkspaceId),
	)
```

Reply:
obskit.StartTime is not present.

Reply:
If something is not there, u can use something liek: logger.NewStringField("xyz", xyz))